lib/generators/deimos/schema_class_guardfile_generator.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
```

Comment:
I'm really not sure we need the Guardfile. Schemas change really rarely and if they change and people are using the record objects, it's likely they'll notice it pretty quickly.

Reply:
good point, can look into removing it :)